src/newMockStore.ts

```diff
@@ -0,0 +1,21 @@
+import { ActionContext, Commit, Dispatch } from 'vuex';
+
+export default function newMockStore(
```

Comment:
we can add a docblock here mentioning what can be passed in? or redundant?

Reply:
hmm probably docs would be redundant in case of typescript with proper tooling, I suppose

Reply:
I think we can get away without a docblock given the ubquity of the structure of a vuex store and the type hinting we take from vuex for the more complex parts. Would you think we need a hint to the vuex docs?